contentcuration/contentcuration/management/commands/get_channel_stats.py

```diff
@@ -0,0 +1,91 @@
+import csv
```

Comment:
Run this to get stats used to determine average channel size for determining big, medium, and small channels